cmd/cmd.go

```diff
@@ -1162,7 +1170,8 @@ func NewCLI() *cobra.Command {
 
 	runCmd.Flags().String("keepalive", "", "Duration to keep a model loaded (e.g. 5m)")
 	runCmd.Flags().Bool("verbose", false, "Show timings for response")
-	runCmd.Flags().Bool("insecure", false, "Use an insecure registry")
+	runCmd.Flags().Bool("insecure", false, "Do not verify registry TLS")
```

Comment:
Also, doesn't this break the current behavior? If I'm used to this flag only changing `https` to `http` and now it does not do that, and instead skips TLS verification without rewriting the scheme, that is not the behavior I would expect. I would want it to push to the http endpoint or none at all, not skip TLS verification.

If that is not what is happening, my bad, but maybe we need to rethink the docs here. If it is what is happening, then: 

I'd like to preempt any "Insecure is insecure" counterpoints with: "This is a breaking change."